charts/consul/templates/server-statefulset.yaml

```diff
@@ -220,8 +229,14 @@ spec:
               CONSUL_FULLNAME="{{template "consul.fullname" . }}"
 
               {{- if .Values.global.secretsBackend.vault.enabled }}
+              {{- if .Values.global.gossipEncryption.secretName }}
               GOSSIP_KEY=`cat /vault/secrets/gossip.txt`
               {{- end }}
+              {{- if .Values.server.serverCert.secretName }}
+              cat /vault/secrets/servercert | jq -r ".certificate" > /vault/secrets/tls.crt
+              cat /vault/secrets/servercert | jq -r ".private_key" > /vault/secrets/tls.key
```

Comment:
reminder to update to make consul-template template these as described in https://github.com/hashicorp/consul-template/blob/master/examples/vault-pki.md#multiple-output-files